src/index.ts

```diff
@@ -100,7 +102,11 @@ export class Client {
     const metadata = {
       project: [this.project],
       collection: [this.collection],
-      "user-agent": [USER_AGENT],
+      "user-agent": [
+        `${USER_AGENT}${
+          this.userAgent !== "" ? ` (via ${this.userAgent})` : ""
```

Comment:
I don't think this needs the `(`s or `via`. So if `this.userAgent` is set, it would be this, for example:

```
sajari-sdk-js/2.0.0-alpha.18 sajari-sdk-react/1.0.0
```

i.e. we just join multiple UAs together with a space.

And if `this.userAgent` is empty, it will just be:

```
sajari-sdk-js/2.0.0-alpha.18
```

Also, the code might possibly read better with something like (untested):

```
"user-agent": [USER_AGENT, this.userAgent].filter(Boolean).join(" ")
```

Reply:
Agree with Jon. Sorry, this was my fault using the wrong format in the original ticket.